rcljava/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ if(NOT ANDROID)
 endif()
 include(UseJava)
 include(JavaExtra)
+include(cmake/add_source_jar.cmake)
```

Comment:
This function seems generally useful. Maybe we should consider moving it to a more common package (e.g. rcljava_common). Then maybe we can also use it for generated code for message types too. What do you think?